src/services/symbolDisplay.ts

```diff
@@ -174,7 +174,7 @@ namespace ts.SymbolDisplay {
             }
 
             let signature: Signature | undefined;
-            type = isThisExpression ? typeChecker.getTypeAtLocation(location) : typeChecker.getTypeOfSymbolAtLocation(symbol.exportSymbol || symbol, location);

```

Comment:
The first line of getTypeOfSymbolAtLocation does `symbol = symbol.exportSymbol || symbol`, so we don't need to do it here.